userguide/services.rst

```diff
@@ -263,41 +250,41 @@ summarizes the available options.
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | Domain                  | string         | Capitalized domain name.                                                                                                  |
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | Server Roll             | drop-down menu | At this time, the only supported role is as the domain controller for a new domain.                                       |
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | DNS Forwarder           | string         | IP address of DNS forwarder. Required for recursive queries when *SAMBA_INTERNAL* is selected.                            |
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | Forest Level            | drop-down menu | Choices are *2000*, *2003*, *2008*, *2008_R2*, *2012*, or *2012_R2*. Refer to                                             |
+   |                         |                | `Understanding Active Directory Domain Services (AD DS) Functional Levels                                                 |
+   |                         |                | <https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc754918(v=ws.10)>`__. |
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | Administration Password | string         | Password to be used for the :ref:`Active Directory` administrator account.                                                |
```

Comment:
Should the field name change?

Reply:
That's what it is in the new UI - I'll open a ticket to change it back and leave a note to update the line when the ticket is resolved.